src/templates/events/helpers/index.js

```diff
@@ -29,3 +29,22 @@ export default event => {
   }
   return `${startDay} ${startMonth} - ${endDay} ${endMonth} ${year} • ${startTime} - ${endTime}`
 }
+
+function filterByDate(event) {
+  if (!this) return true
+  const startDate = moment(event.node.startTime).format('YYYY-MM-DD')
```

Comment:
if we're going to be using the 'YYYY-MM-DD' format across the site, then we should place it in a constants file that we can access across the application,

Reply:
So far its only referenced in this function, i've refactored to pass a variable  that can be shared in the function. However, if we need it elsewhere in the application we can move to a constants file